src/api.ts

```diff
@@ -3,10 +3,15 @@ interface ReviewsParams {
   pageSize: number;
   orderBy: string;
 }
+const baseURL = "https://panda-market-api.vercel.app/products";
```

Comment:
각각의 통신요청 함수에서 하드코딩 된 `url`을 쓰는 대신 상수로 선언해주신 부분 좋습니다. 👍

Reply:
다만, product 경로로 한정되어 있어서 auth 쪽 요청에는 쓰지 못하는게 조금 아쉽습니다.
이런식으로 되도록 하나의 상수로 모든 요청을 컨트롤할 수 있도록 하면 좋을 것 같아요!
```suggestion
const baseURL = "https://panda-market-api.vercel.app";
const productURL = `${baseURL}/products`
const authURL = `${baseURL}/auth`
```